DM-48647: Update doc to include info about ghcr (#3)
lsst-dm/dmtn-218: index.rst

Components
==========

Jenkins (rubin-ci.slac.stanford.edu)
-----------------------

> Control (the "jenkins-master" node) is hosted in a SLAC and Google Cloud Platform; this serves the Jenkins UI at https://rubin-ci.slac.stanford.edu.

Reviewer: ```suggestion
Control (the "jenkins-master" node) is hosted in SLAC (or soon Google Cloud Platform); this serves the Jenkins UI at https://rubin-ci.slac.stanford.edu (or https://ci.lsst.cloud).
```

> Workers are "agent-ldfc" pods running on the GKE Kubernetes (K8s) cluster.

Reviewer: Add the aarch pods here.

They make up a StatefulSet called "agent-ldfc".

> (These pods may have been configured using Terraform somewhere but almost certainly have had many manual changes not reflected in the original Terraform.)

Reviewer: Isn't this line obsolete?

Each pod is composed of a docker-in-docker container (https://github.com/lsst-dm/docker-dind/), a docker-gc container that cleans up old docker containers and images (https://github.com/lsst-dm/docker-docker-gc), and the main swarm container (https://github.com/lsst-dm/docker-jenkins-swarm-client) that actually communicates with the Jenkins central control and executes pipelines.

> Workers are also installed on macOS machines located in the Chile AURA headquarters machine room.

Reviewer: ```suggestion
Workers are also installed on macOS machines located in the Chile AURA headquarters data center.
```

Note that the build-newinstall job builds the version of the rubin-env environment that is specified in etc/scipipe/build-matrix.yaml, not the default in newinstall itself.
The container is pushed with a tag containing that version, as well as a "latest" tag that is typically enabled.

scipipe

> ------

Reviewer: Make the length the same as the text above.
